scrunch/tests/test_accounts.py

```diff
@@ -0,0 +1,108 @@
+# coding: utf-8
+
+import json
+from requests import Response
+from scrunch.accounts import Account
+from scrunch.datasets import Project
+from pycrunch.shoji import Entity, Catalog, View
+
+from .mock_session import MockSession
+
+
+class TestAccount:
+    def make_session(self):
+        root_url = "http://host/api/"
+        account_url = "http://host/api/account/"
+        projects_url = "http://host/api/account/projects/"
+        run_url = "http://host/api/account/run/"
+
+        session = MockSession(site_url=root_url)
+        root_resource = Catalog(session, **{
+            "element": "shoji:catalog",
+            "self": root_url,
+            "index": {},
+            "catalogs": {},
+            "views": {"account": account_url}
+        })
+        act_resource = Entity(session, **{
+            "element": "shoji:entity",
+            "self": account_url,
+            "body": {
+                "name": "test account",
+                "id": "00001",
+            },
+            "catalogs": {
+                "projects": projects_url
+            },
+            "views": {
+                "run": run_url
+            }
+        })
+        run_resource = View(session, **{
+            "element": "shoji:view",
+            "self": run_url,
+            "value": {}
+        })
+        session.add_fixture(account_url, act_resource)
+        session.add_fixture(root_url, root_resource)
+        session.add_fixture(run_url, run_resource)
+        return session
+
+    def test_get_current(self):
+        api_root = self.make_session().root
+        current_act = Account.current_account(api_root)
+        assert current_act.name == "test account"
+        assert current_act.id == "00001"
+        assert isinstance(current_act, Account)
+
+    def test_run_script(self):
+        session = self.make_session()
+
+        response = Response()
+        response.status_code = 204
+
+        session.add_post_response(response)
+        current_act = Account.current_account(session.root)
+
+        current_act.run_script("NOOP;")
```

Comment:
`AttributeError: 'Account' object has no attribute 'run_script'`

Reply:
Eek, left over from rename. It's now called `.execute` changed